src/main/org/nlogo/headless/ChecksumsAndPreviews.scala

```diff
@@ -41,6 +41,8 @@ object ChecksumsAndPreviews {
   }
 
   object Previews {
+    def needsManualPreview(previewCommands: String) =
+      previewCommands contains "need-to-manually-make-preview-for-this-model"
```

Comment:
surely this is always false? should be `containsSlice`, not `contains`

two cheers for Scala's non-typesafe `contains` method